students/ann/storage/login/script.js

```diff
@@ -0,0 +1,11 @@
+const userName = document.querySelector('#userName')
+const userPassword = document.querySelector('#userPw')
+const sendButton = document.querySelector('#submitButton')
+
+sendButton.addEventListener('click', e => {
```

Comment:
this should be a form with submit event. Inside event.target you will have the inputs to get the values